internal/testdata/test-config.json

```diff
@@ -9,7 +9,7 @@
   "Sinks": [
     {
       "PackageRE": "^example.com/core$",
-      "MethodRE": "Sink"
+      "MethodRE": "Sinkf?$"
```

Comment:
Why this change?  Reverting this fails no tests.

Reply:
Hmm. On my machine it breaks a bunch of tests. Specifically, `core.Sinkf` doesn't get identified as a Sink, so it doesn't get reported. Not sure why these were passing before.

Reply:
As far as regexp goes, `Sink` should match `Sinkf` without the `$` anchor...

Reply:
My bad, my explanation was wrong. The problem is that because "Sink" functions are no longer analyzed, the analyzer doesn't look at the bodies of the test functions under `sinks/tests.go` (which have `Sink` in their names) so it misses all the reports. Making the regex more precise allows us to analyze these functions and also catch the sinks.

Therefore the change to the regex is actually to prevent false *negatives*, not false *positives* as I originally mentioned. You are correct w.r.t. regex matching.

Reply:
Ahh, against the `TestSinkEtCetera`.  Gotcha.